package.json

```diff
@@ -89,6 +89,7 @@
         "@open-wc/polyfills-loader": "^0.3.2",
         "@open-wc/testing": "^2.3.3",
         "@open-wc/testing-karma": "^3.1.24",
+        "@spectrum-css/commons": "^2.0.0",
```

Comment:
Please add this to the `styles` package, rather than at the root.

It might make sense to move some of the scripting there over the course as well, but that's something we can get into later.